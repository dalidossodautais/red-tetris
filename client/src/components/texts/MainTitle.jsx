import styled from "styled-components";

export default styled.h1.attrs((props) => ({
  color: props.theme.primaryTextColor,
}))`
  color: ${(props) => props.color};
  text-overflow: ellipsis;
`;
