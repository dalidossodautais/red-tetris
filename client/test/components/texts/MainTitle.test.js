import React from "react";
import renderer from "react-test-renderer";
import MainTitle from "../../../src/components/texts/MainTitle";

describe("MainTitle", () => {
  test("Standard", () => {
    const component = renderer.create(<MainTitle />);
    // rendering
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
    // re-rendering
    tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });
});
