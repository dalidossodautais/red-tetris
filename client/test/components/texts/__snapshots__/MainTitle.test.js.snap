// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainTitle Standard 1`] = `
<h1
  className="sc-bdfBwQ jYASYV"
/>
`;

exports[`MainTitle Standard 2`] = `
<h1
  className="sc-bdfBwQ jYASYV"
/>
`;
